src/components/annual-report/report-data/utils/utils.js

```diff
@@ -0,0 +1,35 @@
+const getRowData = ({ headers, row }) => {
+    const rowData = {}
+    for (let i = 0; i < headers.length; i++) {
+        const header = headers[i]
+        const columnName = header.name
+        const columnValue = row[i]
+        // Add the key-value pair to the row data object
+        rowData[columnName] = columnValue
+    }
+    return rowData
+}
+
+export const mapSectionFourResponse = ({ headers, rows }) => {
```

Comment:
this maps to a `{[dx]: {[ou]: {[pe]: [value]}}}` structure. 

Here the periods should all be current period and for the overall response there should only be one organisation unit, but I thought for consistency purposes, it's probably to keep data in one structure and just provide the period/orgUnit for retrieval from this data structure.